helpers/deploymentConfig.ts

```diff
@@ -109,6 +129,23 @@ const preconfiguredAddresses = {
     VBNB_CorePool: "0xA07c5b74C9B40447a954e1466938b865b6BBea36",
     SwapRouter_CorePool: "0x8938E6dA30b59c1E27d5f70a94688A89F7c815a4",
   },
+  sepolia: {
+    VTreasury: "0xD51008eAe81475b542638433a617f5f4171b39ee",
```

Comment:
According to [this](https://github.com/VenusProtocol/venus-protocol/pull/345/files#diff-85350e55c6b91ff1b61466a0b8a1dc4d6a451222fc1e289e5068b711e0b5d0bfR2), `0x444d9CB57C77Bd262AF33a12f7acC8bdAeB105c9` seems to be the latest Venus Treasury deployed to Sepolia

In the last version, there are a couple of event fields indexed. Not critical changes, but I would use it